package.json

```diff
@@ -9,7 +9,7 @@
     "test": "tap -t360 --no-cov -b test/*.js",
     "snap": "cross-env TAP_SNAPSHOT=1 npm test",
     "posttest": "npm run report",
-    "clean": "rimraf ./.nyc_output ./node_modules/.cache ./.self_coverage ./test/fixtures/.nyc_output ./test/fixtures/node_modules/.cache ./self-coverage",
+    "clean": "rimraf ./.nyc_output ./node_modules/.cache ./.self_coverage ./test/fixtures/.nyc_output ./test/fixtures/node_modules/.cache ./test/fixtures/cli/foo-cache ./test/temp-dir-nyc-integration ./self-coverage",
```

Comment:
Instead of `./test/temp-dir-nyc-integration` lets use `./test/temp-dir-*`.  The same temp-dir pattern is used by test/config-override.js, will potentially be used by other test files in the future.